lib/tasks/uploads.rake

```diff
@@ -362,6 +534,52 @@ def clean_up_uploads
   puts "Done!"
 end
 
+################################################################################
+#                         report missing uploads                               #
+################################################################################
+
+# This might be used to repair after various failures
+task "uploads:report_missing_uploads" => :environment do
```

Comment:
@davidtaylorhq Can I get your eyes on this? 🙂 (the name of the task and what it does)